src/libsyncengine/propagation/executor/executorworker.cpp

```diff
@@ -1107,6 +1107,10 @@ ExitInfo ExecutorWorker::checkLiteSyncInfoForEdit(SyncOpPtr syncOp, const SyncPa
                             syncOp->affectedNode()->size(),
                             syncOp->affectedNode()->id().has_value() ? *syncOp->affectedNode()->id() : std::string(), error);
                     syncOp->setOmit(true); // Do not propagate change in file system, only in DB
+                    // TODO: Vfs functions should return an ExitInfo struct
```

Comment:
Would the comment better placed on line 1108, besides the call to _vfsUpdateMetadata_?

Reply:
Done